docs/CHANGELOG.rst

```diff
@@ -0,0 +1,3 @@
+CHANGELOG
+---------
+.. include:: ../CHANGELOG.md
```

Comment:
```suggestion
.. mdinclude:: ../CHANGELOG.md
```
If you include this and `m2r2` in the test_requirements file, it should fix the incompatibilities between markdown and rst.

Reply:
Yup!  pysatSpaceWeather has an example of this.

Reply:
Thanks for the tip.